tools/portlet_spring_mvc_tmpl/docroot/WEB-INF/spring-context/portlet-application-context.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:aop="http://www.springframework.org/schema/aop"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:mvc="http://www.springframework.org/schema/mvc"
+       xsi:schemaLocation="
+       http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+       http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-3.0.xsd
+       http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd
+       http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc-3.0.xsd">
+
+
+    <!--Enables @Autowired annotation-->
+    <context:annotation-config/>
+
+    <!-- Spring MVC VIEW Configuration -->
+    <bean id="viewResolver" class="org.springframework.web.servlet.view.InternalResourceViewResolver">
+        <property name="viewClass" value="org.springframework.web.servlet.view.JstlView"/>
+        <property name="contentType" value="text/html;charset=UTF-8" />
+        <property name="prefix" value="/WEB-INF/jsp/" />
```

Comment:
This is a tricky thing to do. Form security and clean code perspective the jsp files needs always be hidden, because they wont work when called directly. For our PortletMVC it make sans to make the jsp callable because there is no real controller that prepares data (it's the overall old strut arppoach) and the data are fetched in scriplets in jsps.
Opposing to that you should not have any data fetching in the view and every call to business layer should be done trough Controller class. So you should not be able to call the jsp because they will not work.
Shell I move it or can it stay?